### DISCUSSION TITLE
Search isn't working

### DISCUSSION THREAD

**Discussion by callmefredcom** [transitive-bullshit/nextjs-notion-starter-kit#393] [Q&A]
Nice project, I've implemented it for my site https://scrib.am which used to be on Super. Just one problem though: the Search isn't working (so I've hidden the search widget via css...). It can't communicate with Notion. I've noticed the same issue on this deployment: https://61ec3fa483f671464f0b44f4--amazing-chandrasekhar-c12ace.netlify.app/ Impossible to use Search. At first I thought it was caused by cross-origin issues (before moving to my custom domain) but it's not the cause. Could you help debug this? Thanks. 

Also: I would like to know how to add a custom menu to the header (if it's not too complex).

**Comment by JIeJaitt** [comment]
Hi, Excuse me, do you know how to achieve this function? No matter what I try I can't get this to work！
<img width="874" alt="image" src="https://user-images.githubusercontent.com/77219045/208232078-047b507b-0b30-463d-95d5-876f0316dfdd.png">
